models/spring-ai-watsonx-ai/src/main/java/org/springframework/ai/watsonx/api/WatsonxAiRequest.java

```diff
@@ -62,7 +63,7 @@ public WatsonxAiRequest withProjectId(String projectId) {
     public static Builder builder(String input) { return new Builder(input); }
 
     public static class Builder {
-        private final String input;
+        public static final String MODEL_PARAMETER_IS_REQUIRED = "Model parameter is required";private final String input;
```

Comment:
This looks awkward. 
Please run ./mvnw spring-javaformat:apply

Reply:
Right, apologies, I did the build and it passed so I thought the code linting was right